### ISSUE TITLE
Visibility must be declared on property when attribute added to the Property

### ISSUE THREAD

**Issue by JerryMSunny** [squizlabs/PHP_CodeSniffer#3211]
When attribute is used on Class property the sniffer gives false positive error : Visibility must be declared on property

**Code sample**
```php
#[Attribute(Attribute::TARGET_PROPERTY)]
class Key
{
}

class Student
{
    #[Key]
    private string $rollNumber;

    public function someFunction
    {
        ...
    }
}
```

**To reproduce**
1. Create a file called `Student.php` with the code sample above...
2. Run `phpcs Student.php`
3. See error message displayed
```
Visibility must be declared on property $rollNumber;
```

**Versions (please complete the following information):**
 - OS: Ubuntu 18.04
 - PHP: 8.0
 - PHPCS: 4.0.x-dev
 - Standard: PSR12

**Comment by jrfnl** [comment]
@JerryMSunny PHP 8.0 attributes are not yet supported in PHPCS - see #3182. There is an open PR to fix this - #3203.

I have just tested your code sample against the open PR and that PR would fix the above issue as reported.

However, it also highlights that potentially some sniffs may need adjusting for attributes once the above mentioned PR has been merged. For example, the `PSR1.Files.SideEffects` sniff would start throwing an error after #3203 would be merged, while it shouldn't.

**Comment by gsherwood** [comment]
Can confirm that merging the attribute PR has fixed this issue, so closing it. If you have additional use cases that are not fixed, please report them.
